deploy/infrastructure/dependencies/terraform-commons-dss/default_latest.tf

```diff
@@ -1,5 +1,5 @@
 locals {
-  rid_db_schema = var.desired_rid_db_version == "latest" ?  "4.0.0" : var.desired_rid_db_version
-  scd_db_schema = var.desired_scd_db_version == "latest" ?  "3.1.0" : var.desired_scd_db_version
-  image = var.image == "latest" ? "docker.io/interuss/dss:v0.6.0" : var.image
+  rid_db_schema = var.desired_rid_db_version == "latest" ? "4.0.0" : var.desired_rid_db_version
+  scd_db_schema = var.desired_scd_db_version == "latest" ? "3.1.0" : var.desired_scd_db_version
+  image         = var.image == "latest" ? "docker.io/interuss/dss:v0.6.0" : var.image
```

Comment:
Latest is now v0.7.0 :)

We should probably update RELEASE.md on how to make a release if that requires an additional commit to update this value (and possibly others).  Also, I actually use the GitHub UI to make a release rather than manually pushing a tag.

Reply:
I fixed it here for now. Would it make sense to always push and update along with the version another `latest` tag so it is available from docker hub and do not require updating the code ? I am happy to open another PR for that.

Reply:
Yes, I think that would be great